tests/drivers/spi/spi_loopback/src/spi.c

```diff
@@ -531,6 +602,11 @@ void test_spi_loopback(void)
 
 	LOG_INF("SPI test slow config");
 
+	if (spi_complete_multiple(spi_slow, &spi_cfg_slow)
+	    ) {
```

Comment:
Extra line break to be removed.

Reply:
done, adding this testcase if  CONFIG_SPI_STM32_DMA